applications/openshift/networking/routes_protected_by_tls/rule.yml

```diff
@@ -38,17 +40,16 @@ severity: medium
 
 warnings:
 - general: |-
-    {{{ openshift_cluster_setting("/apis/route.openshift.io/v1/routes?limit=500") | indent(4) }}}
+    {{{ openshift_filtered_cluster_setting({'/apis/route.openshift.io/v1/routes': jqfilter}) | indent(4) }}}
```

Comment:
Should we keep the query string for paging limit responses?

Reply:
I am not too sure about this one